lib/location/method_gnss.c

```diff
@@ -701,6 +701,10 @@ static void method_gnss_positioning_work_fn(struct k_work *work)
 
 	/* Configure GNSS to continuous tracking mode */
 	err = nrf_modem_gnss_fix_interval_set(1);
+	if (err == -EINVAL) {
+		LOG_WRN("First nrf_modem_gnss API function failed. "
+			"It could be that modem's system mode doesn't allow GNSS usage.");
```

Comment:
System or functional mode.

There's a libmodem PR which changes how the statuses from GNSS are mapped, we're looking if we could have a separate error code for this case in the future.

Reply:
Text updated